draft-trammell-rains-protocol.md

```diff
@@ -122,25 +122,25 @@ Its architecture ({{architecture}}) and information model
 System. However, it does take several radical departures from DNS as presently
 defined and implemented:
 
-- Delegation from a superordinate zone to a subordinate zone is done solely
-  with cryptography: a superordinate defines the key(s) that are valid for
-  signing assertions in the subordinate during a particular time interval.
-  Assertions about names can therefore safely be served from any infrastructure.
+- Delegation from a superordinate zone to a subordinate zone is done solely with
+  cryptography: a superordinate defines the key(s), created by the subordinate,
+  that are valid for signing assertions in the subordinate during a particular
+  time interval. Assertions about names can therefore safely be served from any
+  infrastructure.
 - All time references in RAINS are absolute: instead of a time to live, each
   assertion's temporal validity is defined by the temporal validity of the
   signature(s) on it.
 - All assertions have validity within a specific context. A context determines
   the rules for chaining signatures to verify validity of an assertion. The
-  global context is a special case of context, which uses chains from the
-  global naming root key. The use of context explicitly separates global usage
-  of the DNS from local usage thereof, and allows other application-specific
-  naming constraints to be bound to names; see {{assertion-context}}.
-  Queries are valid in one or more contexts, with specific rules for
-  determining which assertions answer which queries; see
+  global context is a special case of context, which uses chains from one or
+  several global naming root key(s). The use of context explicitly separates
```

Comment:
As far as RAINS itself is concerned (in this document) there's one global root; the split root belongs in the rains-scion doc. Will fix after merge.

Reply:
rolled back